MoltenVK/MoltenVK/GPUObjects/MVKInstance.mm

```diff
@@ -345,6 +346,41 @@
 	}
 }
 
+// If the VK_EXT_layer_settings extension is enabled, initialize the local
+// MVKConfiguration from the global version built from environment variables.
+void MVKInstance::initMVKConfig(const VkInstanceCreateInfo* pCreateInfo) {
+
+	if ( !_enabledExtensions.vk_EXT_layer_settings.enabled ) { return; }
+
+	_mvkConfig = mvkConfig();
+
+	VkLayerSettingsCreateInfoEXT* pLSCreateInfo = nil;
+	for (const auto* next = (VkBaseInStructure*)pCreateInfo->pNext; next; next = next->pNext) {
+		switch (next->sType) {
+			case VK_STRUCTURE_TYPE_LAYER_SETTINGS_CREATE_INFO_EXT:
+				pLSCreateInfo = (VkLayerSettingsCreateInfoEXT*)next;
+				break;
+			default:
+				break;
+		}
+	}
+
+	if ( !pLSCreateInfo ) { return; }
+
+	for (uint32_t lsIdx = 0; lsIdx < pLSCreateInfo->settingCount; lsIdx++) {
+		const auto* pSetting = &pLSCreateInfo->pSettings[lsIdx];
```

Comment:
VkLayerSettingEXT contains pLayerName. Ensure that the layer is MoltenVK.

Applications can also set configure the validation layers, extension layers, etc.

Reply:
Ah. Correct. Thanks for catching that. Fixed now.